lib/util/date.ts

```diff
@@ -10,3 +12,12 @@ export function getElapsedDays(timestamp: string): number {
 export function getElapsedMinutes(date: Date): number {
   return Math.floor((new Date().getTime() - date.getTime()) / ONE_MINUTE_MS);
 }
+
+export function getElapsedHours(time: Date | string): number {
+  const pastTime =
+    typeof time === 'string'
+      ? DateTime.fromISO(time)
+      : DateTime.fromJSDate(time);
```

Comment:
Do these throw if the string is not a valid date?

Reply:
It doesn't throw by default but we can change the settings to make it throw.  
```
Settings.throwOnInvalid = true;
```
The `DateTime` object has an `invalid` property which can also be used instead of throwing.

Reply:
I meant that you're not catching it, so if it throws then it's probably a problem. It returns current time if the string passed to it is invalid?

Reply:
it doesn't throw by default and returns an invalid date object which is checked below

Reply:
I added the  below check after rarkins pointed to it😅